charts/stable/firefly-iii/values.yaml

```
@@ -0,0 +1,462 @@
+#
+# IMPORTANT NOTE
+#
+# This chart inherits from our common library chart. You can check the default values/options here:
+# https://github.com/k8s-at-home/library-charts/tree/main/charts/stable/common/values.yaml
+#
+
+image:
+  # -- image repository
+  repository: fireflyiii/core
+  # -- image pull policy
+  pullPolicy: IfNotPresent
+  # -- image tag
+  tag: version-5.6.14
+
+strategy:
+  type: Recreate
+
+cronjobs:
+  # Enable cronjob for creating recurring transactions
```

```suggestion
  # -- Enable cronjob for creating recurring transactions
```
Any reason you havent gone with the `# --` style for all comments?  These allow the comments to be detected as field comments & render in helm-docs.  All your comments in the file are without the `--` which would generate a fairly empty doc.
(Ref: https://github.com/norwoodj/helm-docs)